### PR TITLE
Migration

### PR DESCRIPTION
**Pull request by JunTaoLuo**

### CODE REVIEW

File: src/Microsoft.AspNetCore.ResponseCaching/Streams/ResponseCachingStream.cs [hunk 1]
```diff
@@ -183,19 +183,12 @@ public override void WriteByte(byte value)
             }
         }
 
-#if NETSTANDARD1_3
-        public IAsyncResult BeginWrite(byte[] buffer, int offset, int count, AsyncCallback callback, object state)
```
Comment (anurse) [inline src/Microsoft.AspNetCore.ResponseCaching/Streams/ResponseCachingStream.cs]: Is this right? Wouldn't you want this code and not the other one? This branch was the one used by the `netstandard` build (which was actually the only build anyway)

Comment (Tratcher) [inline src/Microsoft.AspNetCore.ResponseCaching/Streams/ResponseCachingStream.cs]: Yes, we need to override Begin/End stream methods. These were not in early standards.

**Comment by mikeharder** [comment]
Merged to dev: https://github.com/aspnet/ResponseCaching/commit/f536c023f08fc42297e703340639b231424023de

**Comment by mikeharder** [comment]
@JunTaoLuo: Please delete branch

**Pull request merged**